static/app/components/badge/groupPriority.tsx

```diff
@@ -266,12 +272,16 @@ const DropdownButton = styled(Button)`
   box-shadow: none;
 `;
 
-const StyledTag = styled(Tag)`
+const StyledTag = styled(Tag)<{bigTag: boolean}>`
```

Comment:
I swear I tried for so long to not use dynamic props here but I could not figure out what the syntax was since its also combined with a nested selector 🫠 

If you know how to not use dynamic props here pls lmk